es-core/src/GuiComponent.h

```diff
@@ -45,7 +45,7 @@ class GuiComponent
     void setSize(const Eigen::Vector2f& size);
     void setSize(float w, float h);
     virtual void onSizeChanged() {};
-	
+
```

Comment:
How do you do what ? Open a new PR ? I don't have time to help with using git, sorry.